### DISCUSSION TITLE
Where player stores starred playlists and collection?

### DISCUSSION THREAD

**Discussion by JonDVstRm** [strawberrymusicplayer/strawberry#919] [Q&A] [answered]
Hi, yeah, I admit I am lazy to search this time, so maybe it is duplicate. But with the other player on which you are based on (Clementine of course) I often lost my starred playlist, which was the gems, the best of the best. The only way no to loose them was to save them periodicly (hmm maybe I can do some script) and sync them with cloud. 

So maybe you can save me some work and tell me where are the starred playlist stored, so I can sync them directly, also to know where is the database will be helpfull (collection).

Thx, appreciate your work, cheers

**Comment by jonaski** [comment, marked as answer]
In the sqlite database: `~/.local/share/strawberry/strawberry/strawberry.db`
